staging/src/k8s.io/apimachinery/pkg/runtime/converter.go

```diff
@@ -229,7 +229,7 @@ func fromUnstructured(sv, dv reflect.Value) error {
 	case reflect.Interface:
 		return interfaceFromUnstructured(sv, dv)
 	default:
-		return fmt.Errorf("unrecognized type: %v", dt.Kind())
+		return fmt.Errorf("Can't convert value %#v of type %v -> %v", sv, st.Kind(), dt.Kind())
```

Comment:
Errors are typically not capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings